src/toil/batchSystems/abstractGridEngineBatchSystem.py

```diff
@@ -417,11 +433,11 @@ def issueBatchJob(self, command: str, jobDesc, job_environment: Optional[Dict[st
             else:
                 gpus = jobDesc.accelerators
```

Comment:
This code looks like it was maybe left in and is a function now (self.count_needed_gpus)?